admin_guide/managing_networking.adoc

```diff
@@ -1246,6 +1246,67 @@ IP address. If `egressIPs` is set on a `NetNamespace`, but there is no node
 hosting that egress IP, then egress traffic from the namespace will be dropped.
 endif::openshift-origin,openshift-enterprise[]
 
+[[admin-guide-automatic-egress-ip]]
+=== Enabling Automatic Egress IPs
+
+Similar to xref:enabling-static-ips-for-external-project-traffic[Enabling Static
+IPs for External Project Traffic], as a cluster administrator you can assign
+egress IPs to namespaces by setting the `egressIPs` field to the `NetNamespace`
+resource. With fully-automatic egress IPs, you can set the `egressCIDRs` field
+of each node's `HostSubnet` resource to indicate the range of egress IPs that can
+be hosted. Namespaces that have requested egress IPs are matched with nodes that
+are able to host those egress IPs, then the egress IPs are assigned to those nodes.
+
+High Availability is automatic, which means if a node hosting egress IPs goes down
+and there are nodes that are able to host those egress IPs, based on the `egressCIDRs`
+and `HostSubnet` values, then the egress IPs will move to a new node. When the
```

Comment:
"based on the egressCIDRs and HostSubnet values" isn't quite right. "based on the egressCIDRs values of the HostSubnet resources" would be better